From: Mike Hommey <mh@glandium.org>
Date: Thu, 25 Feb 2010 12:42:51 +0100
Subject: [PATCH] Header adjustment for imported changes

Changes from bz#506433 rely on bz#471877 being applied, and we now need
gtkversion.h on top of gtkcontainer.h.
---
 mozilla/widget/src/gtk2/mozcontainer.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/mozilla/widget/src/gtk2/mozcontainer.h b/mozilla/widget/src/gtk2/mozcontainer.h
index 7135873..51850ea 100644
--- a/mozilla/widget/src/gtk2/mozcontainer.h
+++ b/mozilla/widget/src/gtk2/mozcontainer.h
@@ -40,6 +40,7 @@
 #define __MOZ_CONTAINER_H__
 
 #include <gtk/gtkcontainer.h>
+#include <gtk/gtkversion.h>
 
 #ifdef __cplusplus
 extern "C" {
-- 
